src/models/db/migrations/20240808125209_query_tables_additional_column.ts

```diff
@@ -0,0 +1,15 @@
+import type { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('query', (def) => {
+    def.uuid('query_id_for_response').defaultTo(null)
```

Comment:
I'm thinking we should also add a column for query role so We can tell if we've asked the question or been asked. Maybe something like `role` with an enum with values `requester` and `responder`

Reply:
I thought we can know this based on whether we have received the 'query-id-for-response' == a query_id we arebeing asked to respond to

Reply:
We probably can, but this is getting to the point where it's difficult to tell and I feel these sorts of inferences require explicit knowledge of how the system works. A column for this would be useful and simpler. We can also filter on it in the UI then and in the event handler much more simply

Reply:
I shall add the column

Reply:
do you think we should add a defaultTo('requester') value? or make it nullable? 

```ts
// currently
    def
      .enum('role', ['requester', 'responder'], {
        useNative: true,
        enumName: 'query_role',
      })
      .notNullable()
```